lib/stripe.js

```diff
@@ -133,6 +135,26 @@ Stripe.prototype = {
     );
   },
 
+  setAppInfo: function(info) {
+    if (info && typeof info !== 'object') {
+      throw new Error('AppInfo must be an object.');
+    }
+
+    // Kill the cached UA string because it may no longer be valid
+    Stripe.USER_AGENT_SERIALIZED = undefined;
+
+    this._appInfo = APP_INFO_PROPERTIES.reduce(function(accum, prop) {
```

Comment:
I was thinking _at least `name`_.

Here's the stripe-ruby method signature:

``` ruby
def self.set_app_info(name, version: nil, url: nil)
```

As you can see we've tried to force presence on `name` and kept `version` and `url` as additional flavoring if you're so inclined.